frontend/src/queries/nodes/DataTable/renderColumnMeta.tsx

```diff
@@ -67,7 +57,21 @@ export function renderColumnMeta(key: string, query: DataTableNode, context?: Qu
             />
         )
     } else {
-        title = queryFeatures.has(QueryFeature.selectAndOrderByColumns) ? extractExpressionComment(key) : key
+        const extractedKey = extractExpressionComment(key)
```

Comment:
Using the extracted key means queries can look like `any(properties) -- context.columns.error` but the query context can still look like:
```
const context: QueryContext = {
        columns: {
                error: {
                        width: '50%',
                        render: CustomGroupTitleColumn,
                },
        },
    }
```

Before it had to be:
```
const context: QueryContext = {
        columns: {
                'any(properties) -- Error': {
                        width: '50%',
                        render: CustomGroupTitleColumn,
                },
        },
    }
```

Reply:
Why not use just column aliases? This "comment extraction" is a leftover from early days when the HogQL parser was actually a Python parser (`import ast`) and didn't support most SQL syntax. I then implemented this `--` hack. Now we should just use `any(properties) as "Any property"`